### PR TITLE
Tono branch

### PR DESCRIPTION
**Pull request by tonkomisko**
hi, tono, please check tono's code. I' would like to merge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,46 +1,201 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<!-- <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"> -->
```
Comment (gunkooo) [inline index.html]: remove this part, it's unused

**Review by gunkooo — CHANGES_REQUESTED** [review]
please see my comment

File: index.html (hunk 1 above)
Comment (tonkomisko) [reply index.html]: already changed, please check my change

**Review by tonkomisko — COMMENTED** [review]
done and done

**Pull request merged**